nncf/quantization/advanced_parameters.py

```diff
@@ -197,6 +198,9 @@ class AdvancedQuantizationParameters:
         the calibration dataset, then in case batch_size of the data source > 1 batchwise_statistics sets to True,
         otherwise sets to False.
     :type batchwise_statistics: Optional[bool]
+    :param conservative_quantizers_merging: If true quantizers with different granularity (per-tensor and per-channel)
```

Comment:
What do you think about the following parameter name "strict_propagation"?
cc' @AlexKoff88 @MaximProshin

Reply:
done